src/scripts/table.js

```diff
@@ -0,0 +1,137 @@
+let tableMode = document.querySelector('.table_mode');
+let switchModeLeft = document.querySelector('.mode_switch_left');
+let switchModeRight = document.querySelector('.mode_switch_right');
+let countryMode = "World"
+
+let country = document.querySelector('.country')
+let cases = document.querySelector('.cases')
+let deaths = document.querySelector('.deaths')
+let recovered = document.querySelector('.recovered')
+
+
+let modes = ['Total', 'Today', 'Total per 100.000 population', 'Today per 100.000 population'];
+
+let currMode = tableMode.innerHTML;
+let currStat;
+
+switchModeLeft.addEventListener("click", switchLeft);
+
+switchModeRight.addEventListener("click", switchRight);
+
+
+function switchLeft() {
+    if (modes.indexOf(currMode) === 0) {
+        tableMode.innerHTML = modes[3]
+        currMode = tableMode.innerHTML;
+        createCountryRow(currStat);
+    } else {
+        tableMode.innerHTML = modes[modes.indexOf(currMode) - 1];
+        currMode = tableMode.innerHTML;
+        createCountryRow(currStat);
+    }
+}
+
+function switchRight() {
+    if (modes.indexOf(currMode) === 3) {
+        tableMode.innerHTML = modes[0];
+        currMode = tableMode.innerHTML;
+        createCountryRow(currStat);
+    } else {
+        tableMode.innerHTML = modes[modes.indexOf(currMode) + 1];
+        currMode = tableMode.innerHTML;
+        createCountryRow(currStat);
+    }
+}
+
+function createCountryRow(stat) {
+    if (currMode === 'Total') {
+        country.innerHTML = stat.country;
+        cases.innerHTML = stat.cases;
+        deaths.innerHTML = stat.deaths;
+        recovered.innerHTML = stat.recovered;
+        currStat = stat;
+    } else if (currMode === 'Today') {
+        country.innerHTML = stat.country;
+        cases.innerHTML = stat.todayCases;
+        deaths.innerHTML = stat.todayDeaths;
+        recovered.innerHTML = stat.todayRecovered;
+       currStat = stat;
+    } else if (currMode === 'Total per 100.000 population') {
+        country.innerHTML = stat.country;
+        cases.innerHTML = Math.round((stat.cases / stat.population) * 100000);
+        deaths.innerHTML = Math.round((stat.deaths / stat.population) * 100000);
+        recovered.innerHTML = Math.round((stat.recovered / stat.population) * 100000);
+        currStat = stat;
+    } else if (currMode === 'Today per 100.000 population') {
+        country.innerHTML = stat.country;
+        cases.innerHTML = Math.round((stat.todayCases / stat.population) * 100000);
+        deaths.innerHTML = Math.round((stat.todayDeaths / stat.population) * 100000);
+        recovered.innerHTML = Math.round((stat.todayRecovered / stat.population) * 100000);
+        currStat = stat;
+    }
+}
+
+async function getInf(url = 'https://disease.sh/v3/covid-19/countries', url1 = 'https://corona.lmao.ninja/v2/countries', country) {
+
+    let ctr = ''
+    country && country !== 'World' ? ctr = country : false
+    ctr ? countryMode = ctr : false
```

Comment:
` country && country !== 'World' ? ctr = country : false`
 ` ctr ? countryMode = ctr : false`
code is difficult for understanding